src/Autofac.Integration.ServiceFabric/Autofac.Integration.ServiceFabric.csproj

```diff
@@ -2,8 +2,9 @@
 
   <PropertyGroup>
     <Description>Autofac integration for Azure Service Fabric. Provides service factory implementations for Actors, Stateful Services and Stateless Services.</Description>
-    <VersionPrefix>3.0.0</VersionPrefix>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+    <!-- VersionPrefix patched by AppVeyor -->
+    <VersionPrefix>0.0.1</VersionPrefix>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net472</TargetFrameworks>
```

Comment:
I don't think we need to target 472 as well if we target netstandard2.0; 472 and ns2.0 should be fully compatible (unlike net461).

Reply:
Good catch. Removed `net472` targets.